apps/server/src/modules/task-card/controller/dto/task-card.response.ts

```diff
@@ -31,12 +32,12 @@ export class TaskCardResponse {
 	})
 	cardElements?: CardElementResponse[];
 
-	@ApiPropertyOptional()
+	@ApiProperty()
 	@DecodeHtmlEntities()
-	courseName?: string;
+	courseName!: string;
```

Comment:
that `!` is not needed here if you initialize the value in constructor, which in my opinion should be done as we have the course as a required value